### ISSUE TITLE
Version 3.12 requirement

### ISSUE THREAD

**Issue by SoftologyPro** [TemporalLabsLLC-SOL/TemporalPromptGenerator#3]
Does the setup really need 3.12 of Python?
If that requirement was removed could it work with Python 3.10.x?
I have hundreds of other scripts supported in Visions of Chaos and they all use 3.10.x.

eg, could the -3.12 argument be removed?
```
Running command: py -3.12 -m venv D:\Tests\Temporal Prompt Engine\TemporalPromptGenerator\WINDOWS\CogVideo\inference\gradio_composite_demo\CogVx
Python 3.12 not found!
Installed Pythons found by py Launcher for Windows
 -3.10-64 *
```

**Comment by SoftologyPro** [comment]
Editing out `"-3.12",` in SETUP.py line 271 gets the install finishing fine with Python 3.10.x
GUI now starts fine.  Could you remove the -3.12 argument?

**Comment by SoftologyPro** [comment]
Do you have a quick tutorial on how to use it?
UI starts.  I type a prompt.  I click Generate Video Prompts and it asks for a folder.  I point to a new empty folder, but nothing gets created there.  I click Generate Videos and it asks for a prompts list.  Assuming this is what it did not create before?

**Comment by TemporalLabsLLC-SOL** [comment]
> Do you have a quick tutorial on how to use it? UI starts. I type a prompt. I click Generate Video Prompts and it asks for a folder. I point to a new empty folder, but nothing gets created there. I click Generate Videos and it asks for a prompts list. Assuming this is what it did not create before?

Yes! I'm recording that tomorrow now that I'm happy with the newest video prompt formatting. :)

**Comment by TemporalLabsLLC-SOL** [comment]
> Editing out `"-3.12",` in SETUP.py line 271 gets the install finishing fine with Python 3.10.x GUI now starts fine. Could you remove the -3.12 argument?

It is important to have access to both versions on your system. The `RUN-FIRST-PRE-FLIGHT-CHECK` should help guide you through the pre-flight requirements.

The setup.py creates two UNIQUE environments though and that's why both are needed. One for the main app and generating sound effects and a secondary to allow video generation scripts. 

I'm working on a full exe installer but I have to figure out a few things before that will be out and I'm doing this full bootstraps. I'll put the exe on my website when it happens.
